source/Connection/AMDSThreadedTCPDataServer.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include <QCoreApplication>
 
-#include "source/util/AMDSErrorMonitor.h"
+#include "util/AMErrorMonitor.h"
```

Comment:
Just so I understand this correctly, now that you are merging this code with acquaman proper, you are switching to using AM's error mon rather than using your duplicate error mon?